### PR TITLE
Add Ember Europe Q2 2023

### PR DESCRIPTION
**Pull request by emmasofiah2o**
- Add event
- Add description
- Add image

### CODE REVIEW

**Comment by emmasofiah2o** [comment]
Screenshot 2023-05-16 at 12 14 08

File: src/calendar/2023-06-15-embereurope-dublin.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+title: "Ember Europe Q2 Meetup"
+image: "/assets/images/calendar/2023-06-15-embereurope-dublin/embereurope.png"
+location: Dublin & remote
+url: https://www.meetup.com/ember-europe/
+kind: meetup
+color: purple
+---
+
+The second Ember Europe meetup of 2023 is happening on June 15th and is hosted
+by Intercom at their Dublin's office. The Ember Europe meetup series brings
```
Comment (marcoow) [inline src/calendar/2023-06-15-embereurope-dublin.md]:
```suggestion
by Intercom at their Dublin office. The Ember Europe meetup series brings
```

**Review by marcoow — CHANGES_REQUESTED** [review]
one typo

**Comment by marcoow** [comment]
@emmasofiah2o please ping reviewers when change requests have been adressed – this went unnoticed for 2 weeks now…

**Comment by emmasofiah2o** [comment]
> @emmasofiah2o please ping reviewers when change requests have been adressed – this went unnoticed for 2 weeks now…

I thought you would be automatically notified since I committed your suggestion. I'll tag you next time.

**Pull request merged**